### PR TITLE
Add missing info on packages (Chocolatey, Guix, SlackBuilds)

### PR DESCRIPTION
**Pull request by AMDmi3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -404,18 +404,25 @@ You can read about [getting started](https://julialang.org/manual/getting-starte
 
 The following distributions include julia, but the versions may be out of date due to rapid development:
 
+<a href="https://repology.org/metapackage/julia">
+    <img src="https://repology.org/badge/vertical-allrepos/julia.svg" alt="Packaging status" align="right">
```
Comment (tkelman) [inline README.md]: when there are multiple entries for the same name (like gentoo, nixpkgs stable) this image is only showing one of them

Comment (AMDmi3) [inline README.md]: Yes, it's showing the lastest version available for a specific repo. Showing all of them would make it just too big, and availability of legacy versions are not really interesting for most users.

Comment (tkelman) [inline README.md]: those specific examples appear to have both a "dev" version that looks like some random build of nightly / master, and a release version - but only the dev version is showing up in the svg which isn't as useful

Comment (AMDmi3) [inline README.md]: Unfortunately, there's no way to know whether a version is development or an official one. So it's just showing the latest one. There's one improvement planned which will hide gentoo's "9999" version, but for nixpkgs there's no way to fix it.

Comment (tkelman) [inline README.md]: I'm fine with linking to the summary page but the image would be a bit misleading then.

Comment (AMDmi3) [inline README.md]: I'm ok with that. Should I update the PR?

Comment (tkelman) [inline README.md]: we can wait a bit, see if anyone else has opinions

**Comment by fredrikekre** [comment]
#23484

**Pull request closed without merging**